pkg/drivers/kic/oci/network.go

```diff
@@ -70,8 +81,25 @@ func dockerGatewayIP(profile string) (net.IP, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "get network bridge")
 	}
+	networksOutput := strings.TrimSpace(rr.Stdout.String())
+	networksSlice := strings.Fields(networksOutput)
+	// Look for the minikube container within each docker network
+	for _, net := range networksSlice {
+		// get all containers in the network
+		rs, err := runCmd(exec.Command(Docker, "network", "inspect", net, "-f", "'{{range $k, $v := .Containers}}{{$v.Name}} {{end}}'"))
+		if err != nil {
+			return nil, errors.Wrapf(err, "get containers in network")
+		}
+		containersSlice := strings.Fields(rs.Stdout.String())
```

Comment:
Call strings.TrimSpace on rr.Stdout.String(): otherwise it will leave the final newline in.

Reply:
The newLine character is already trimmed by the strings.Fields() function.

Reply:
I assumed it was too, and in fact, I tried do duplicate this on Go Playground, and it behaves as you mention. I suspcet my confusion stems from my earlier debugging attempt which the quoting fixed:

```
12.984928  519897 network.go:68] container "'minikube" does not match profile "minikube"
12.984954  519897 network.go:68] container "'" does not match profile "minikube"
13.027496  519897 network.go:68] container "''" does not match profile "minikube"
```